aspnet/migration/aspnetcoremigration.rst

```diff
@@ -0,0 +1,190 @@
+Migrating from ASP.NET to ASP.NET Core
+======================================
+
+In this article:
+
+.. contents:: Sections
+  :local:
+  :depth: 1  
+
+Overview
+--------
+
+This migration guide will cover the essentials on how to migrate projects from ASP.NET to ASP.NET Core. 
+
+For information on migrating from DNX to .NET CLI, see the `DNX RC1->RC2 migration guide <http://placeholder>`_. 
```

Comment:
see `` `Migrating from DNX to .NET Core CLI <http://dotnet.github.io/docs/core-concepts/servicing.html>`__ ``